content/docs/hooks-rules.md

```diff
@@ -38,7 +38,7 @@ npm install eslint-plugin-react-hooks --save-dev
 {
   "plugins": [
```

Comment:
The [URL](https://github.com/facebook/react/blame/main/packages/eslint-plugin-react-hooks/README.md#L21-L30 ) you sent has  `"extends"` but here you have modified in the `"plugins"` right ? 

Snippet in the URL:
```js
{
  "extends": [
    // ...
    "plugin:react-hooks/recommended"
  ]
}
```

Reply:
Good catch, thanks.